### ISSUE TITLE
Logitech Control Center

### ISSUE THREAD

**Issue by jenham** [dokterdok/Continuity-Activation-Tool#226]
The CAT works great, but if I installed Logitech's Control Center it tells me I have no wifi hardware installed. Any recommendations? I'm using a late 2011 MBP with a 4.0 Bluetooth card.

**Comment by sysfloat** [comment]
Did you use the latest version (2.9.3) with El Capitan support?
Does it mess with the bootargs? Run nvram -p in terminal.
Does it mess with the IO802Family.kext? Compare before and after installing Logitech Control Center.

**Comment by jenham** [comment]
Yes I have that version installed.
What should the nvram results look like?
I don't believe it does, the time that the files were last modified stays the same after LCC install.

**Comment by sysfloat** [comment]
There should be something like "boot-args	kext-dev-mode=1" in there.

**Comment by jenham** [comment]
Yes i do see that line. I haven't had the problem after uninstalling and reinstall both of these so it might be gone, but if its an issue I will report it and hopefully we can figure it out! Thanks!

**Comment by sysfloat** [comment]
Oh okey, good to hear.
